lib/api/users/user_service.dart

```diff
@@ -29,11 +31,25 @@ class UserService {
   /// [usersLookup], a method that allows you to convert user IDs into full user
   /// objects in bulk.
   ///
```

Comment:
Would be nice to have descriptions for the parameters in the docs, just like the other methods do.